src/cmap/commands.ts

```diff
@@ -6,6 +6,7 @@ import type { Long, Document, BSONSerializeOptions } from '../bson';
 import type { ClientSession } from '../sessions';
 import type { CommandOptions } from './connection';
 import { MongoRuntimeError, MongoInvalidArgumentError } from '../error';
+// import { deserialize } from '../bson';
```

Comment:
I'd say after the revert we can just remove this now.

Reply:
Oh, good catch!